resources/views/models/blog/categories/form.blade.php

```diff
@@ -53,6 +53,13 @@ class="-ml-1 mr-2 h-5 w-5"
 
             />
 
+            <x-fab::forms.checkbox
+                id="should_index"
+                label="Should be indexed"
+                wire:model="category.indexable"
+                help="If checked this will allow search engines (i.e. Google or Bing) to index the page so it can be found when searching on said search engine."
```

Comment:
```suggestion
                help="If checked this will allow search engines (i.e. Google or Bing) to index the category page so it can be found when searching on said search engine."
```